README.md

```diff
@@ -64,4 +64,4 @@ OR
 uvicorn requestor.asgi:app --reload
 ```
 
-Try out the API at: <http://localhost:8000/docs>.
+Try out the API at <http://localhost:8000/_status> or <http://localhost:8000/docs> (you might have to set `DOCS_URL_PREFIX` to `""` in your configuration file for the docs endpoint to work).
```

Comment:
1. We don't get the API when we hit [<http://localhost:8000/_status>](url) right? I only see a `{status:OK}` message. 
2.  > you might have to set `DOCS_URL_PREFIX` to `""`

    Why is it **might** ? Is it not going to be the same all the time?

Reply:
_status is an endpoint of the API

Reply:
2. it used to work without it, and we didn't make changes, so i'm not sure what caused it. i don't have time to investigate but just updating the readme might help the next person